client/lib/js/react/components/accounts.js

```diff
@@ -1,14 +1,10 @@
 import React from 'react';
-
-import { Row } from 'react-bootstrap/lib';
-
+import { safeInvoke } from 'js-utility-belt/es6';
 import classnames from 'classnames';
 
 import AccountActions from '../actions/account_actions';
 import AccountStore from '../stores/account_store';
 
-import BigchainDBLedgerPlugin from './bigchaindb_ledgerplugin';
-
 import Spinner from './spinner';
 
 const AccountList = React.createClass({
```

Comment:
Since the main export of this file is `AccountList`, it should be renamed to `account_list.js`.